package.json

```diff
@@ -20,6 +20,7 @@
     "build:main": "cross-env NODE_ENV=production webpack --bail --config webpack.config.main.js",
     "build:renderer": "cross-env NODE_ENV=production webpack --bail --config webpack.config.renderer.js",
     "start": "electron src --disable-dev-mode",
+    "clean": "rm -rf dist/ release/ node_modules/ src/node_modules/ build/ src/*_bundle.js src/**/*_bundle.js",
```

Comment:
`dist` and `build` are not used in this project. Is it necessary explicitly removing them?

Reply:
@yuya-oc: I thought I saw that some updates didn't get picked up because those folders already existed. If this is wrong, we should remove those from the clean task.

Reply:
@dmeza Yeah, please remove. Those folders were actually used in past versions, so I think that you possibly saw them.